react/features/base/participants/reducer.ts

```diff
@@ -63,10 +63,12 @@ const PARTICIPANT_PROPS_TO_OMIT_WHEN_UPDATE = [
 
 const DEFAULT_STATE = {
     dominantSpeaker: undefined,
-    everyoneIsModerator: false,
     fakeParticipants: new Map(),
     local: undefined,
     localScreenShare: undefined,
+    numberOfNonModeratorParticipants: 0,
```

Comment:
I think we need to reset these when you leave the conference, otherwise when joining a breakout room for example they won't get updated to the right value.

Reply:
Hmm... I will test this but I would expect to get participant left action for every participant when the user leaves a room which should effectively reset them, isn't it?